Moli/Core/Screens/Profile/ProfileView.swift

```diff
@@ -209,116 +221,104 @@ struct ProfileView: View {
+                        )
+
+                    }
+
                     if let biography = biography() {
                         if !biography.isEmpty {
                             HStack {
-                                Text(biography + "Test")
-                                    .font(.body)
+                                Text(biography).font(.labelSmall)
                             }
                         }
                     }
-                    
-                    if let websiteURL = websiteURL(), !websiteURL.isEmpty {
-                        HStack {
-                            Image(systemName: "link")
-                            Link(websiteURL, destination: URL(string: websiteURL)!)
-                                .font(.subheadline.weight(.medium))
-                                .textCase(.lowercase)
+                    HStack (spacing: 12) {
+                        if let websiteURL = websiteURL(), !websiteURL.isEmpty {
+                            Button (
+                                action: { openLink(link: websiteURL) },
+                                label: {
+                                    Image("Website")
+                                        .frame(width: 20, height: 20)
+                                })
+                                .padding(12)
+                                .background(Color.gray50)
+                                .clipShape(Circle())
                         }
-                    }
-                    
-                    if let joinDate = joinDate(), !joinDate.isEmpty {
-                        HStack(spacing: 4) {
-                            Image(systemName: "calendar")
-                            Text("Joined \(joinDate)")
-                                .font(.subheadline)
-                                .foregroundColor(.gray600)
+
+                        if let instagramLink = getInstagramProfileLink() {
+                            Button (
+                                action: { openLink(link: instagramLink) },
+                                label: {
+                                    Image("Instagram")
+                                        .frame(width: 20, height: 20)
+                                })
+                                .padding(12)
+                                .background(Color.gray50)
+                                .clipShape(Circle())
                         }
                     }
-                    
-                    HStack {
-                        HStack(spacing: 4) {
-                            Text(followers())
-                                .fontWeight(.semibold)
-                                .foregroundColor(.primary)
-                            Text("Followers")
-                                .foregroundColor(.gray600)
-                        }
-                        
+
+
+                    if isMe() {
                         HStack(spacing: 4) {
```

Comment:
:art: Rent UX mässigt kommer det här beteendet ge `EditProfileView()` en paddingTrailing av 4px, även fast det är en EmptyView i sig själv så kommer den att räknas in som element, vilket i sin tur påverkar layout:en av knappen. I dey här fallet var det bara 4 extra pixlar, men det är mer som en hands up inför framtiden

```suggestion
                        HStack(spacing: 0) {
```

Reply:
Du kan testa det här genom att ha spacing: 40, så kommer du se vad jag menar